blog/_posts/2019-01-19-light-graphs.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "GSoC 2018: Parallel Graph Development"
```

Comment:
Parallel Implementations of Graph Analysis Algorithms?

Reply:
That's a better name. 
I used "GSoC 2018: Parallel Graph Development" because it is the official name of the GSoC project.

Should I use "GSoC 2018: Parallel Implementations of Graph Analysis Algorithms"

Reply:
Yes the name suggested by Chris is slightly better

Reply:
Done!